eng/scripts/Initialize-VcpkgRelease.ps1

```diff
@@ -38,29 +31,29 @@ param (
 # If there's nothing in the "port" folder to upload set SkipVcpkgUpdate to true
 # and exit. Other steps will check SkipVcpkgUpdate to decide whether to move
 # forward.
-if (!(Get-ChildItem -Path "$SourceDirectory/port")) {
+if (!(Get-ChildItem -Path "$SourceDirectory/port/CONTROL")) {
     Write-Host "###vso[task.setvariable variable=SkipVcpkgUpdate]true"
     exit
 }
 
 $packageSpec = Get-Content -Raw -Path $PackageSpecPath | ConvertFrom-Json
-$assetFilename = "$($packageSpec.packageName).tar.gz"
+$tarGzUri = "https://github.com/$GitHubRepo/archive/$($packageSpec.packageName).tar.gz" 
+
+Write-Host "Downloading tarball to compute hash from $tarGzUri" 
+$localTarGzPath = New-TemporaryFile
+Invoke-WebRequest -Uri $tarGzUri -OutFile $localTarGzPath
+
+$sha512 = (Get-FileHash -Path $localTarGzPath -Algorithm SHA512).Hash.ToLower()
+Write-Host "SHA512: $sha512"
 
-# Upload the asset to the release
-Write-Verbose "Uploading asset: $assetFilename..."
```

Comment:
Do we really wish to no longer upload the port file to the release?

Reply:
Correct. We do not need to upload assets to the release. The release gives us the .tar.gz asset that we end up using for free.